lib/logstash/outputs/scalyr.rb

```diff
@@ -76,6 +76,9 @@ class LogStash::Outputs::Scalyr < LogStash::Outputs::Base
   config :flat_tag_prefix, :validate => :string, :default => 'tag_'
   config :flat_tag_value, :default => 1
 
+  # Whether or not to convert Bigint values to strings to avoid JSON encoding errors
```

Comment:
Probably a good idea to add some context - why we need this (bug in jruby, which version is it fixed in) and also link to that github issue